globalConfig.json

```diff
@@ -0,0 +1 @@
+{"mongoUri":"mongodb://127.0.0.1:40083/"}
```

Comment:
Ca sert à quoi exactement ?

Reply:
C'est créé automatiquement lors des tests. En fait pour les tests je créé une database "in-memory" pour ne pas aller taper une db "réelle" et le fait de faire ça ça me crée ce fichier : https://dev.to/paulasantamaria/testing-node-js-mongoose-with-an-in-memory-database-32np